src/frontend/src/btc/services/btc-send.services.ts

```diff
@@ -1,24 +1,37 @@
 import type { UtxosFee } from '$btc/types/btc-send';
 import { convertNumberToSatoshis } from '$btc/utils/btc-send.utils';
+import type { SendBtcResponse } from '$declarations/signer/signer.did';
 import { selectUserUtxosFee } from '$lib/api/backend.api';
+import { sendBtc as sendBtcApi } from '$lib/api/signer.api';
+import { ProgressStepsSendBtc } from '$lib/enums/progress-steps';
 import type { BtcAddress } from '$lib/types/address';
 import { mapToSignerBitcoinNetwork } from '$lib/utils/network.utils';
+import { waitAndTriggerWallet } from '$lib/utils/wallet.utils';
 import type { Identity } from '@dfinity/agent';
 import type { BitcoinNetwork } from '@dfinity/ckbtc';
 
 const DEFAULT_MIN_CONFIRMATIONS = 6;
 
+type BtcSendServiceParams<T = unknown> = {
```

Comment:
Not sure if it needs to be generic in this particular case?

Reply:
No need, I removed it.